test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -26,16 +27,25 @@ public DatabaseLoaderTests(ITestOutputHelper output)
         [LightGBMFact]
         public void IrisLightGbm()
         {
-            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                // https://github.com/dotnet/machinelearning/issues/4156
-                return;
-            }
-
             var mlContext = new MLContext(seed: 1);
 
-            var connectionString = GetConnectionString(TestDatasets.irisDb.name);
-            var commandText = $@"SELECT * FROM ""{TestDatasets.irisDb.trainFilename}""";
+            string connectionString;
```

Comment:
This seems to be repeated code copy pasted. Can you please refactor this better?

Reply:
I originally did this here to make the connection string and the command text more understandable, and to highlight the difference between accessing a MSSQL database and a SQLite database. I will change this block of code with the below to remove the implicit declaration of `connectionString` and `commandText`.

```csharp
            DatabaseSource databaseSource;
            // Non-Windows builds do not support SqlClientFactory/MSSQL databases. Hence, an equivalent
            // SQLite database is used on Linux and MacOS builds.
            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                databaseSource = new DatabaseSource(
                    SqlClientFactory.Instance,
                    GetConnectionString(TestDatasets.irisDb.name),
                    $@"SELECT * FROM ""{TestDatasets.irisDb.trainFilename}""");
            else
                databaseSource = new DatabaseSource(
                    SQLiteFactory.Instance,
                    GetSQLiteConnectionString(TestDatasets.irisDbSQLite.name),
                    $@"SELECT * FROM ""{TestDatasets.irisDbSQLite.trainFilename}""");

            var loader = mlContext.Data.CreateDatabaseLoader<IrisVectorData>();

            var trainingData = loader.Load(databaseSource);
```